api/Document.json

```diff
@@ -9521,13 +9599,13 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/requestStorageAccess",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "85"
             },
             "chrome_android": {
-              "version_added": false
+              "version_added": "85"
             },
             "edge": {
-              "version_added": false
+              "version_added": "85"
```

Comment:
I thought Edge had shipped `document.requestStorageAccess` before Chrome, but maybe not. They were directly involved though: https://blogs.windows.com/msedgedev/2020/07/08/introducing-storage-access-api/

Reply:
MS is driving the implementation in chromium: https://bugs.chromium.org/p/chromium/issues/detail?id=989663

They enabled it to be on by default in Edge before Chrome but I can’t find any publicly source that indicates which Edge version it was enabled in 😿